PDBminer

```diff
@@ -814,15 +770,18 @@ def get_all_discrepancies(df):
     warnings : Qualitative warinings regarding the structure, list of strings. 
 
     """
-    print("FUNCTION: get_all_discrepancies(df)")
+    logging.debug("FUNCTION: get_all_discrepancies(df)")
 
     warnings = []
         
     df = df[df.uniprot_seq != "-"]
     df = df[df.pdb_seq != "-"]        
 
     if len(df[df["uniprot_seq"] != df["pdb_seq"]]) != 0:
-        all_mutations = df[df["uniprot_seq"] != df["pdb_seq"]].apply(lambda row: f"{row['uniprot_seq']}{row['uniprot_pos']}{row['pdb_seq']}", axis=1).tolist()
+        #all_mutations = df[df["uniprot_seq"] != df["pdb_seq"]].apply(lambda row: f"{row['uniprot_seq']}{row['uniprot_pos']}{row['pdb_seq']}", axis=1).tolist()
```

Comment:
you can remove this line instead of commenting it (there is no reason to keep it since it will stay in the git history)